site/profile/templates/cvmfs/config.d.epp

```diff
@@ -0,0 +1,8 @@
+# Create repo to store alien cache instead of having them created/store in default.local file
```

Comment:
This file should be named `alien_cache.conf.epp` instead of `config.d.epp`

Reply:
Done